url.py

```diff
@@ -0,0 +1,63 @@
+import hashlib
+
+class URL:
+    # key: long_url, value: URL()
+    URL_DICT = {}
```

Comment:
This shouldn't be capitalized because it's not a constant. You modify it throughout the program.